### DISCUSSION TITLE
How to get previous node eLements data by clicking on node element?

### DISCUSSION THREAD

**Discussion by Azim-Ahmed** [xyflow/xyflow#1435] [Q&A]
When I click on node element , I need to know the previous element node's data.

**Comment by albatahmet** [comment]
Hello Azim-Ahmet,
My english may be bad, sorry.

`{
        id: '2',
        data: {label: 'test1', label2:'test2'},
        position: {x: 100, y: 125},
    },`

I think each node can have 2 labels. You can update one of them when connected by typing a command like setLabel from outside.

**Reply by Azim-Ahmed** [reply]
Thanks for trying, I have given this section to the backend , but this would be better if I make it on frontend.You can connect with me azimaahmed36@gmail.com

**Reply by Azim-Ahmed** [reply]
I find out the way, your'e wrong.
